services/frontend/src/store/slices/notification.slice.js

```diff
@@ -0,0 +1,16 @@
+import {createSlice} from "redux-starter-kit";
+import {createRequestSlices} from "store/utils";
+import mounts from "store/mountpoints";
+
+export const notificationSlice = createSlice({
+    slice: mounts.notification,
+    initialState: null,
+    reducers: {
+        setNotification: (state, action) => action.payload,
```

Comment:
An improvement to this would be to be able to show an arbitrary number of notifications -- i.e. store a list of notifications of just the one. Otherwise, the user might miss out on some notifications being overridden by others.

Reply:
Maybe make use of https://github.com/iamhosseindhv/notistack? 

It's shown inside of the [docs](https://material-ui.com/components/snackbars/#notistack) for material-ui so it's not some random package.

Reply:
Sure, looks nice.

Reply:
Simple to implement and works really well, I kept `PopupNotifier` as I needed someplace to connect to the store though to actually queue up the snackbars.